docs-src/getting-started.md

```diff
@@ -0,0 +1,55 @@
+# Getting started with Services APIs
+
+## Installing CRDs
+
+Service APIs is a collection of Kubernetes Custom Resource Definitions that need
+to be installed into any Kubernetes cluster.
```

Comment:
Maybe "This project provides a collection of Custom Resource Definitions that can be installed into any Kubernetes 1.16+ cluster."